miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -618,10 +619,10 @@ HotTarget.sample = (function() {
               'slidesConsumed', HotUtils.validator.requiredPositiveInt),
 
           // Tissue Processing: Single Cell columns
-          HotUtils.makeColumnForFloat('Initial Cell Conc.', (show['Tissue Processing'] && config.targetSampleClass.alias == 'Single Cell'),
-              'initialCellConcentration', false),
-          HotUtils.makeColumnForText('Digestion', (show['Tissue Processing'] && config.targetSampleClass.alias == 'Single Cell'),
-              'digestion', {
+          HotUtils.makeColumnForFloat('Initial Cell Conc.', (show['Tissue Processing'] && config.targetSampleClass.alias
```

Comment:
Should this be `makeColumnForDecimal` instead of `...ForFloat`?

Reply:
Yes. Fixed